cobbler/settings/migrations/V3_2_1.py

```diff
@@ -91,6 +91,7 @@
         Optional("nsupdate_tsig_algorithm", default="hmac-sha512"): str,
         Optional("nsupdate_tsig_key", default=[]): [str],
         "power_management_default_type": str,
+        Optional("proxies", default=[]): [str],
```

Comment:
Why do we add those keys to all versions? The only SUMA version that has those keys is 4.3.2 and newer with Cobbler 3.3.3 (plus patches). So why add it to all the much older versions?

Reply:
Actually the `proxies` settings is not new in 4.3.2, it is used since previous versions already at the time of configuring "tftpsync" (susemanager-tftpsync package), which also provides the post cobbler sync trigger for pushing stuff to proxies).

That is why I added this also to all migration files, as we want to preserve this value that might be present already in `settings` file from Cobbler 3.1.2 in the context of SUMA/Uyuni.

Reply:
Yes. So we need this as a maximum - and we don't even need that afaik - starting with 3.1.2. There is no reason to make this gloablly available in all versions.